src/System.Data.SqlClient/tests/Tools/TDS/TDS.EndPoint/SSPI/SecurityWrapper.cs

```diff
@@ -24,7 +24,7 @@ internal class SecurityWrapper
         /// <param name="pvGetKeyArgument">This parameter is not used and should be set to NULL</param>
         /// <param name="phCredential">A pointer to a CredHandle structure to receive the credential handle</param>
         /// <param name="ptsExpiry">A pointer to a TimeStamp structure that receives the time at which the returned credentials expire</param>
-        [DllImport(Interop.Libraries.SspiCli, CharSet = CharSet.Auto)]
+        [DllImport(Interop.Libraries.SspiCli, CharSet = CharSet.Auto, EntryPoint="AcquireCredentialsHandleW")]
```

Comment:
I guess you meant Exact-spelling , it's not currently required since we pass UseDefaultPinvoke to ilc for test runs.